### PR TITLE
Add companies and addresses, remove guests

### PR DESCRIPTION
**Pull request by tfarras**

### CODE REVIEW

File: admin-ui/src/Companies/index.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  Edit,
+  Show,
+  ReferenceManyField,
+  Datagrid,
+  EditButton,
+  TextField,
+  TextInput,
+  required,
+} from 'react-admin';
+import AddAddressButton from './AddAddressButton';
```
Comment (grigoreme) [inline admin-ui/src/Companies/index.js]: import ordering

File: admin-ui/src/Companies/index.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+      </Tab>
+    </TabbedShowLayout>
+  </Show>
+);
+
+const CompanyDescription = ({ record }) => {
+  return (
+    <div
+      /* eslint-disable-next-line react/no-danger */
+      dangerouslySetInnerHTML={
```
Comment (grigoreme) [inline admin-ui/src/Companies/index.js]: is this required?

File: api/src/address/address.controller.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import { AddressEntity } from './address.entity';
+
+@Crud({
+  model: {
+    type: AddressEntity,
+  },
+})
+@Controller('address')
+export class AddressController {
+  constructor(public service: AddressService) {
```
Comment (grigoreme) [inline api/src/address/address.controller.ts]: move } to prev line

File: api/src/address/address.entity.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  street: string;
+
+  @Column()
+  zip: string;
+
+  @Column()
+  notes: string;
+
+  @Column('simple-json')
+  geo: { lng: string, lat: string };
```
Comment (grigoreme) [inline api/src/address/address.entity.ts]: why use here JSON? it could be just two columns

File: api/src/company/company.entity.ts [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import { Entity, Column, PrimaryGeneratedColumn, BaseEntity, AfterLoad, OneToMany, RelationId, ManyToMany, JoinTable } from 'typeorm';
```
Comment (grigoreme) [inline api/src/company/company.entity.ts]: split into multiple lines please

File: api/src/s3/s3.service.ts [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+    this.s3Region = config.get('s3.region');
+    this.s3 = new S3({ region: this.s3Region, signatureVersion: 'v4' });
+  }
+
+  async signIn(req: Request, res: Response): Promise<any> {
+    const { objectName, contentType, path } = req.query;
+    const objectNameChunks = objectName.split('/');
+    const filename = objectNameChunks[objectNameChunks.length - 1];
+    const mimeType = contentType;
+    const fileKey = `${path || ''}/${objectName}`;
```
Comment (grigoreme) [inline api/src/s3/s3.service.ts]: you could make the default value of the path to be the empty string and remove this from inside template string

File: api/src/address/address.entity.ts (hunk 4 above)
Comment (tfarras) [reply api/src/address/address.entity.ts]: because component for react-admin sends data in this format, and it's simpler to store data as JSON.

**Pull request merged**